miner/bid_simulator.go

```diff
@@ -593,6 +603,41 @@ func (b *bidSimulator) simBid(interruptCh chan int32, bidRuntime *BidRuntime) {
 		return
 	}
 
+	// fill transactions from mempool
+	if b.config.EnlargeBid {
+		delay := b.engine.Delay(b.chain, bidRuntime.env.header, &b.delayLeftOver)
+		if delay != nil || *delay > 0 {
+			log.Debug("BidSimulator: stopTimer", "block", bidRuntime.env.header.Number,
+				"header time", time.Until(time.Unix(int64(bidRuntime.env.header.Time), 0)),
+				"commit delay", *delay, "DelayLeftOver", b.delayLeftOver)
+
+			stopTimer := time.NewTimer(0)
+			defer stopTimer.Stop()
+			<-stopTimer.C // discard the initial tick
+
+			stopTimer.Reset(*delay)
+
+			bidTxsSet := hashset.New()
+			for _, tx := range bidRuntime.bid.Txs {
+				bidTxsSet.Add(tx.Hash())
+			}
+
+			bidRuntime.env.gasPool.SubGas(PayBidTxGas)
```

Comment:
The side effect is that, even `SubGas` here and the gasPool is possible go to be zero eventually, the `bidRuntime.commitTransaction(b.chain, b.chainConfig, payBidTx)` will fail too.

Reply:
fixed, add minGas in the params of fillTransactions